pipeline/tests/test_regressions.py

```diff
@@ -10,3 +13,172 @@ def test_issue_0002():
     node = build_fake_node(omcore.Person)
     data = node.to_jsonld()
     assert data["@type"] == "https://openminds.ebrains.eu/core/Person"
+
+
+def test_issue_0003():
+    # https://github.com/openMetadataInitiative/openMINDS_Python/issues/3
+    # validate() does not complain about direct entries that should be lists
+
+    # we address this by always wrapping a single item in a list in such cases
+
+    some_file = omcore.File(
+        iri=IRI("http://example.com/some_file.txt"),
+        name="some_file.txt",
+    )
+
+    node1 = omcore.FileArchive(
+        iri=IRI("http://example.com/archive.zip"),
+        format=omcore.ContentType(name="application/zip"),
+        source_data=[some_file],  # multiple=True, min_items=1
+    )
+    node2 = omcore.FileArchive(
+        iri=IRI("http://example.com/archive.zip"),
+        format=omcore.ContentType(name="application/zip"),
+        source_data=some_file,  # multiple=True, min_items=1
+    )
+    # on export, a single item should be wrapped in a list, where the property expects an array
+    expected = {
+        "@context": {"vocab": "https://openminds.ebrains.eu/vocab/"},
+        "@type": "https://openminds.ebrains.eu/core/FileArchive",
+        "vocab:IRI": "http://example.com/archive.zip",
+        "vocab:format": {
+            "@type": "https://openminds.ebrains.eu/core/ContentType",
+            "vocab:name": "application/zip",
+        },
+        "vocab:sourceData": [
```

Comment:
this should not be embedded but linked (linked object array 1-N; optional)

Reply:
here I'm using the `embed_linked_nodes=True` argument to `to_jsonld()`, which is convenient for testing and for working with JSONLD locally.
when saved to file, `embed_linked_nodes=False` is always used.

Reply:
okay